.github/workflows/integration-tests-sentry-cli.yml

```diff
@@ -0,0 +1,44 @@
+name: integration-tests-sentry-cli
+
+on:
+  push:
+    branches:
+      -master
+  pull_request:
+
+defaults:
+  run:
+    working-directory: integration-test
+
+jobs:
+  integration-test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [macos-latest]
+    steps:
+      - uses: actions/checkout@v3
+      - uses: subosito/flutter-action@v2
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.10.5'
+      - uses: actions/setup-java@v3
+        with:
+          distribution: 'temurin'
```

Comment:
Any reason to be different than others?

Reply:
I had errors on CI with the other distribution.

Reply:
I see, that's strange since we use the other distribution in the Dart repo, but if that unblocks you, all good.